### DISCUSSION TITLE
'plugins': The "mkdocs-jupyter" plugin is not installed #144

### DISCUSSION THREAD

**Discussion by everndah** [squidfunk/mkdocs-material#5418] [Q&A] [answered]
Why do I get this error (See Below) and I have installed mkdocs-jupyter. For this reason the run fails.

![image](https://user-images.githubusercontent.com/66929420/234000545-fb24aeb3-5a37-42fe-8585-e2618e41aae9.png)

**Comment by jaceklaskowski** [comment]
Can you show the other (earlier) steps, with the one where you installed the plugin? It's not installed by default so either you use a custom docker image with the plugin installed or you install it as a github action.

**Reply by everndah** [reply]
Followed this process to install(See link) https://pypi.org/project/mkdocs-jupyter/

**Reply by everndah** [reply]
Created a GitHub Actions workflow, .github/workflows/ci.yml, and pasted the following contents:

![image](https://user-images.githubusercontent.com/66929420/234006594-7606f3c6-7a02-45eb-9f80-e04f5d2b91cc.png)

**Comment by everndah** [comment, marked as answer]
@jaceklaskowski , thanks for the reply . The commit everndah [7ca3229](https://github.com/everndah/python_tutorials/commit/7ca322925a0ff0d8c2c19790f722cd914d36cda3) solved the issue.

Changed from;
 
 ` - run: mkdocs gh-deploy --force`
 `- run: pip install mkdocs-jupyter`
 
 To;
 
 `- run: pip install mkdocs-jupyter`
 `- run: mkdocs gh-deploy --force`
